pkgs/applications/misc/masterpdfeditor/default.nix

```diff
@@ -1,22 +1,26 @@
-{ stdenv, fetchurl, sane-backends, qtbase, qtsvg, nss, autoPatchelfHook }:
+{ stdenv, fetchurl, sane-backends, qtbase, qtsvg, nss, autoPatchelfHook, lib, makeWrapper }:
 
 let
-  version = "5.1.12";
+  version = "5.1.36";
 
 in stdenv.mkDerivation {
   name = "masterpdfeditor-${version}";
 
   src = fetchurl {
     url = "https://code-industry.net/public/master-pdf-editor-${version}_qt5.amd64.tar.gz";
-    sha256 = "1i3pdrhnlj06phm36gs42s6b94pigcfb8wa5dhmplxn0dqp434hq";
+    sha256 = "10kydjg9qi6kvzn7ny5nk4xn2fsm47anq3al89hq97ppm9dlvh48";
   };
 
-  nativeBuildInputs = [ autoPatchelfHook ];
+  nativeBuildInputs = [ autoPatchelfHook makeWrapper ];
 
   buildInputs = [ nss qtbase qtsvg sane-backends stdenv.cc.cc ];
 
   dontStrip = true;
 
+  postInstall = ''
+    wrapProgram $out/bin/masterpdfeditor5 --prefix QT_PLUGIN_PATH : ${qtbase}/lib/qt-5.${lib.versions.minor qtbase.version}/plugins
```

Comment:
Why problem does this wrapper solve? The package builds and runs fine without it on my machine.

Reply:
I encountered the same problem as: https://github.com/NixOS/nixpkgs/issues/24256 and I was able to fix it with this wrapper. What's weird is that we don't have the same error.

Reply:
I didn't need this wrapper with 5.1.12, only with 5.1.36

Reply:
That's a known error when you try to run Qt apps on a machine which has another Qt version installed. I'm not sure if we should add a wrapper to fix this because this may cause other problems. What does our Qt specialist @ttuegel recommend?

Reply:
Wrapping the program like this will surely break it for someone. Failing to wrap the program will also surely break it for someone. There is practically no way to support Qt applications in Nixpkgs in every configuration; something is going to be broken. The only important thing is that it works for you, the maintainer. If you're going to wrap, the `QT_PLUGIN_PATH` entry should be `${lib.getBin qtbase}/${qtbase.qtPluginPrefix}` instead of what is above.

Reply:
ping @cmcdragonkai @flokli